docs.md

```diff
@@ -23,72 +23,82 @@ permalink: /docs/
 
 ## New to Flutter?
 
-* [Write Your First Flutter App](/get-started/codelab/)  
-  A walkthrough on creating a Flutter app that generates names for startup companies.
+Start here.
 
-* [Flutter for Android Developers](/flutter-for-android/)  
-  Tips for Android developers.
+* [Write Your First Flutter App](/get-started/codelab/)<br>
+  Create a Flutter app that generates names for startup companies.
 
-* [HTML/CSS patterns](/web-analogs/)  
-  Tips for web developers showing how HTML and CSS syntax maps to Flutter widgets.
+* [Flutter for Android Developers](/flutter-for-android/)<br>
+  Review these tips if you have Android experience.
 
-* [Building Layouts in Flutter](/tutorials/layout/)  
-  Walkthrough showing how to create layouts in Flutter, where everything is a widget.
+* [HTML/CSS patterns](/web-analogs/)<br>
+  Review these HTML -> Flutter analogs if you have web experience.
 
-* [Add Interactivity to Your Flutter App](/tutorials/interactive/)  
-  Walkthrough showing how to add a stateful widget to your app.
+* [Building Layouts in Flutter](/tutorials/layout/)<br>
+  Learn how to create layouts in Flutter, where everything is
+  a widget.
 
-* [A Tour of the Flutter Widget Framework](/widgets-intro/)  
-  A short, conceptual tour through the Flutter framework.
+* [Adding Interactivity to Your Flutter App](/tutorials/interactive/)<br>
+  Learn how to add a stateful widget to your app.
+
+* [A Tour of the Flutter Widget Framework](/widgets-intro/)<br>
+  Learn more about Flutter's react-style framework.
 
 &nbsp;
 
 
-## Want to skill up?
+## Want to improve your skills?
```

Comment:
I'm leaning towards the original "skill up" but not strongly.

Reply:
I think Seth's point is that non-native English speakers might find that difficult to parse.

Reply:
Yeah, but:

1. I'm non-native and I understood
2. It's not that hard to figure it out
3. It's not like this line is critical to grok